tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -35,9 +35,8 @@ def _get_model_type(self):
         if self.params_aux.get('use_daal', False):
             # Disabled by default because it appears to degrade performance
             try:
-                # TODO: Use sklearnex instead once a suitable toggle option is provided that won't impact future models
                 # FIXME: DAAL OOB score is broken, returns biased predictions. Without this optimization, can't compute Efficient OOF.
-                from daal4py.sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+                from sklearnex.ensemble import RandomForestClassifier, RandomForestRegressor
                 logger.log(15, '\tUsing daal4py RF backend...')
```

Comment:
```suggestion
                logger.log(15, '\tUsing sklernex RF backend...')
```

Reply:
Good suggestion, I have updated this log for RF, KNN, and linear models.